workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -172,14 +172,15 @@ private void DrawCustomSnapshotDir()
             using (new EditorGUILayout.HorizontalScope())
             {
                 EditorGUI.BeginDisabledGroup(true);
-                    EditorGUILayout.TextField(CustomSnapshotPathLabel, toolsConfig.CustomSnapshotPath);
+                EditorGUILayout.TextField(CustomSnapshotPathLabel, PlayerPrefs.GetString("CustomSnapshotPath", "snapshots/default.snapshot"));
                 EditorGUI.EndDisabledGroup();
+
                 if (GUILayout.Button("Open", EditorStyles.miniButtonRight, GUILayout.ExpandWidth(false)))
                 {
-                    string path = EditorUtility.OpenFilePanel("Select snapshot", "", "snapshot");
+                    var path = EditorUtility.OpenFilePanel("Select snapshot", "", "snapshot");
```

Comment:
This still needs to open in the right directory by settings the 2nd argument to the directory of our currently selected snapshot.